source/sass/views/indexpage.scss

```diff
@@ -0,0 +1,18 @@
+
+.term-list {
+  display: inline-block;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+
+  .term {
+    display: inline-block;
+    padding: 0 0.5em;
+    margin-left: 0.25em;
+    border: 1px solid $black;
+    border-radius: 1em;
+    font-size: 0.8em;
```

Comment:
tag should be in`.body-small` with override to make the text black

Reply:
```suggestion
    color: $black;
```